Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1881,7 +1881,7 @@ def insert_cef_alerts_command(client, args):
 
 def isolate_endpoint_command(client, args):
     endpoint_id = args.get('endpoint_id')
-
+    disconnected_should_return_error = argToBoolean(args.get('suppress_disconnected_endpoint_error', True))
```

Comment:
```suggestion
    disconnected_should_return_error = not argToBoolean(args.get('suppress_disconnected_endpoint_error', False))
```
while it is correct that if `suppress_disconnected_endpoint_error` does not exist, then `disconnected_should_return_error` should be true, 
if `uppress_disconnected_endpoint_error` exists, `disconnected_should_return_error` should get the opposite value of it.